index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ko-KR">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Starbucks</title>
+  <link rel="stylesheet" href="./reset.css" />
+  <link rel="stylesheet" href="./index.css" />
+</head>
+<body> 
+  <container class="container"> 
```

Comment:
is there such tag called container in html?

```suggestion
  <div class="container"> 
```